jerry-ext/handler/handler-assert.c

```diff
@@ -17,16 +17,16 @@
 #include "jerryscript-port.h"
 
 /**
- * Assert for scripts. The routine calls jerry_port_fatal on assertion failure.
+ * Hard assert for scripts. The routine calls jerry_port_fatal on assertion failure.
  *
  * @return true - if only one argument was passed and that argument was a boolean true.
  *         Note that the function does not return otherwise.
  */
 jerry_value_t
-jerryx_handler_assert (const jerry_value_t func_obj_val, /**< function object */
-                       const jerry_value_t this_p, /**< this arg */
-                       const jerry_value_t args_p[], /**< function arguments */
-                       const jerry_length_t args_cnt) /**< number of function arguments */
+jerryx_handler_assert_fatal (const jerry_value_t func_obj_val, /**< function object */
```

Comment:
Why this function is not a void function?

Reply:
Because it is an external handler function with a fixed signature that has to take a given number of arguments and must return a `jerry_value_t`. And it does indeed return with jerry-true if the assert passes. Just like it did from the introduction of this function.

Reply:
I forgot to open the full code. Patch is ok but I dont like the if-return else-return constructs. There is no need an else after a return. Less indentation.